### PR TITLE
doc: add README of lua_ssl_certificate and lua_ssl_certificate_key

### PR DESCRIPTION
**Pull request by ZJfans**
I hereby granted the copyright of the changes in this pull request
to the authors of this lua-nginx-module project.

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -3336,6 +3338,44 @@ This directive was first introduced in the `v0.9.11` release.
 
 [Back to TOC](#directives)
 
+lua_ssl_certificate
+-------------------
+
+**syntax:** *lua_ssl_certificate &lt;file&gt;*
+
+**default:** *no*
```
Comment (zhuizhuhaomeng) [inline README.markdown:3346]:
```suggestion
**default:** *none*
```

File: README.markdown [hunk 2]
```diff
@@ -3336,6 +3338,44 @@ This directive was first introduced in the `v0.9.11` release.
...
+See also [lua_ssl_certificate_key](#lua_ssl_certificate_key) and [lua_ssl_verify_depth](#lua_ssl_verify_depth).
+
+[Back to TOC](#directives)
+
+lua_ssl_certificate_key
+-----------------------
+
+**syntax:** *lua_ssl_certificate_key &lt;file&gt;*
+
+**default:** *no*
```
Comment (zhuizhuhaomeng) [inline README.markdown:3365]:
```suggestion
**default:** *none*
```

Comment (zhuizhuhaomeng) [reply README.markdown:3365]: the default value of lua_ssl_trusted_certificate is also wrong, would you please fixed it?

Comment (ZJfans) [reply README.markdown:3365]: > the default value of lua_ssl_trusted_certificate is also wrong, would you please fixed it?

ok, directives without a default value should indeed use none, not no

**Review by zhuizhuhaomeng — APPROVED** [review]

**Pull request merged**